events/tests/test_event_get.py

```diff
@@ -578,6 +578,10 @@ def test_event_status_filter(
     get_list_and_assert_events("event_status=eventrescheduled", [event2])
     get_list_and_assert_events("event_status=eventcancelled", [event3])
     get_list_and_assert_events("event_status=eventpostponed", [event4])
+    get_list_and_assert_events(
+        "event_status=eventscheduled,eventrescheduled,eventcancelled,eventpostponed",
+        [event, event2, event3, event4],
+    )
```

Comment:
In addition to this, I guess we could also test that multiple but not all work (like 2 statuses), as it is trivial to add that test.

Reply:
Added new test for this case.